features/taxation/applying_taxes/applying_correct_taxes_for_order_with_promotion_giving_discount_for_an_item_in_order.feature

```diff
@@ -0,0 +1,50 @@
+@applying_taxes
+Feature: Apply correct taxes on order item level when order has applied a promotion with discount for an item in order
+    In order to pay proper amount when buying goods
+    As a Visitor
+    I want to have correct taxes applied to my order when it has applied a promotion
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And default tax zone is "US"
```

Comment:
Yes, but in this step is setting a default tax zone for a channel and this is required to apply taxes